plugins/adr/package.json

```diff
@@ -58,9 +58,9 @@
     "@material-ui/icons": "^4.9.1",
     "@types/react": "^16.13.1 || ^17.0.0 || ^18.0.0",
     "lodash": "^4.17.21",
-    "react-markdown": "^8.0.0",
+    "react-markdown": "^9.0.0",
     "react-use": "^17.2.4",
-    "remark-gfm": "^3.0.1"
+    "remark-gfm": "^4.0.0"
```

Comment:
According to Knip (https://github.com/backstage/backstage/blob/master/plugins/adr/knip-report.md) these two deps are not actually used in this plugin so we can probably remove these here so the ADR plugin is not applicable to these updates

Reply:
One needs to validate knip though, it is far from perfect still. One thing it misses, for example, is peer dependencies that were added for an actual reason but isn't used as a direct import.

Not saying I'm sure that's the case here. But do check it out carefully in the code etc.

Reply:
Ah shoot I should have known!